fuel-core-interfaces/src/block_importer.rs

```diff
@@ -0,0 +1,11 @@
+/// Currently just placeholder for new block included and new block created events.
+#[derive(Clone, Debug)]
+pub enum NewBlockEvent {
+    /// send this to eth
+    NewBlockCreated(u64),
+    NewBlockIncluded {
+        height: u64,
+        /// height where we are finalizing stake and token deposits.
+        eth_height: u64,
```

Comment:
If this isn't specific to Ethereum, but is rather generic to some L1, then it shouldn't use `eth` in the name. Is there a better name for some sort of generic data availability layer? Maybe `da_height`?

Reply:
`da_height` seems okay.
This is going to be changes when we start deciding on message format between modules.

Reply:
@Voxelot just to note here, i slighty changed idea of `da_height` to represent eth block in past that will enable all token/validator deposits.